Makefile

```diff
@@ -37,6 +37,17 @@ test: validate-manifest lint unit-test cleanup-integration-test integration-test
 unit-test:
 	pytest assisted-events-scrape/tests/unit
 
+ci-integration-test:
+	./tools/generate_mockserver_configmap.sh | kubectl delete -f - || true
+	./tools/generate_mockserver_configmap.sh | kubectl create -f - || true
+	oc apply -f assisted-events-scrape/tests/integration/manifests
+	oc wait --timeout=120s --for=condition=Available deployment --all
+	oc wait --timeout=300s --for=condition=Ready pods --all
+	oc apply -f openshift/
+	oc wait --timeout=30s --for=condition=Available deployment --all
+	oc wait --timeout=30s --for=condition=Ready pods --all
+	ES_SERVER=elasticsearch-master:9200 ES_INDEX=assisted-service-events pytest assisted-events-scrape/tests/integration
```

Comment:
Where is the test executing? From the claimed cluster or in a different cluster? What's the best way to run a test against elasticsearch instance deployed in the claimed cluster?

Reply:
from a pod that has access to the claimed cluster (so yes, a different cluster)
after claiming the cluster, you should have ``KUBECONFIG`` that you can use to setup ES/mock-server/etc.

Reply:
IIUC the deployment should be correct in this PR - since KUBECONFIG would point already to the claimed cluster? However in the test (last line) I am assuming I have connectivity to k8s service: I understand this is not the case? Do I need to expose elasticsearch to check against it? How is this usually done?

Reply:
Right about the first part
About exposing ES, I think that's what we're doing to get actual port of the service to work with: https://github.com/openshift/assisted-service/blob/87616d920e04325a92716144c05976d110ea0ec8/Makefile#L13
(yeah, not that pretty. We can possibly use a simpler version of it)

Reply:
hmm, I'm not sure
I think the best way to be certain is to do a "test" of just long sleep duration, and inspect the cluster we get. There's a way to connect to prow jobs' pods and debug stuff, so it should be possible